api/api/routers/agents_v1.py

```diff
@@ -156,7 +157,12 @@ async def get_agent_stats(
 ) -> Page[AgentStat]:
     from_date = from_date or datetime.now() - timedelta(days=7)
     items = [
-        AgentStat(agent_uid=stat.agent_uid, run_count=stat.run_count, total_cost_usd=stat.total_cost_usd)
+        AgentStat(
+            agent_id=str(stat.agent_uid),
```

Comment:
are you looking for the string agent_id as in `meta-agent` for example ? What you have here would give you a stringified int (`1234`)